src/VisualStudio/Next/Remote/ServiceHubRemoteHostClient.cs

```diff
@@ -16,9 +16,10 @@ namespace Microsoft.VisualStudio.LanguageServices.Remote
 {
     internal partial class ServiceHubRemoteHostClient : RemoteHostClient
     {
-        private readonly HubClient _hubClient;
         private readonly Stream _stream;
-        private readonly JsonRpc _rpc;
+
+        protected readonly HubClient _hubClient;
+        protected readonly JsonRpc _rpc;
```

Comment:
Per offline suggestion by @heejaechang these accessibility changes were not required (just added a string parameter and renamed CreateCodeAnalysisSessionAsync instead).